views/pages/editTask.ejs

```diff
@@ -17,16 +17,11 @@
     </p>
     <form action=<%= "/tasks/update/" + task.id %> method="post">
       <label for="name">Name:</label> <br>
-      <input name="name" value="<%=task.name %>"><br>
-      <label for="complete">Completed: </label>
-      <% if (task.completed) { %>
-        <input type="checkbox" name="complete" value="true" checked><br><br>
-        <% } else { %>
-          <input type="checkbox" name="complete" value="true"><br><br>
-          <% } %>
```

Comment:
Simplify with ternary since the only difference in the two branches is whether the `checked` attribute is included or not